charts/test-guide/README.md

```diff
@@ -31,3 +31,15 @@ To see all configurable options with detailed comments, visit the chart's [value
 # Show chart values
 $ helm show values tracetronic/test-guide
 ```
+
+## Migration
+
+### To 0.4.0
+
+This version supports a configurable TEST-GUIDE application directory which must match the used image.
+
+#### values.yaml
+
+| Name | Description | Value |
+| ---- | ----------- | ----- |
+| `baseSettings.tgWorkspace` | TEST-GUIDE base directory | default: `/home/tg_user/` |
```

Comment:
Add info to '/app/' value to get the previous behavior (reuse existing clusters)